usagov/vote-gov-drupal
# Vote-979 -> Footer Contact Test

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[Vote-979](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-979&assignee=611272e0eb7aef00694b8a97)

## Description (optional)
This Pr will add the first test to the home page.. this will check that the footer contact section of the page is present 

## Deployment and testing (required, if applicable)
### Testing steps
1. pull down branch
2. start local 
3. cd into `testing`
4. run `npm run cy:open`
5. select the `home-page.cy.js` test and verify that test is passing
